test/visibility/private/moduleSymbols/accessPrivateSiblingSubmodule.chpl

```diff
@@ -6,7 +6,7 @@ module parent {
   }
 
   module sibling {
-    import parent.child;
+    import parent.{child}; // use bracketed form instead
```

Comment:
tiny bit confused by this.. are we losing a variant by not adding this as a new test and modifying an existing one instead

Reply:
No, there was an identical test in the same directory already.  This modifies one of the two identical tests so that they cover different things